tstest/integration/nat/nat_test.go

```diff
@@ -445,6 +453,12 @@ func TestEasyEasy(t *testing.T) {
 	nt.want(routeDirect)
 }
 
+func TestJustIPv6(t *testing.T) {
```

Comment:
What about 4+6 on one side and Just6 on the other

Reply:
Done. But these tests don't quite work yet for reasons I haven't yet debugged.

IPv6 works by hand in a gokrazy shell.